proposals/records-wip.md

```diff
@@ -59,6 +59,8 @@ For a record struct or a record class:
 
 * A public get-only auto-property is created. Its value is initialized during construction with the value of the corresponding primary constructor parameter. Each "matching" inherited abstract property's get accessor is overridden.
 
+  * This property is also `initonly`, meaning the backing field can be modified in the [with](#With) expression below, if the corresponding `get` accessor is accessible
```

Comment:
Perhaps 
```suggestion
  * This property also has an `init` accessor, meaning the backing field can be modified in the [with](#With) expression below, if the `init` accessor is accessible
```
given that we're going with `init` being the keyword?

Reply:
Out of scope for now, I'm just trying to make records work, not user-declared `initonly` accessors.